[2025-10-02T07:40:37.293+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T07:40:37.299+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T07:40:35.330938+00:00 [queued]>
[2025-10-02T07:40:37.301+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T07:40:35.330938+00:00 [queued]>
[2025-10-02T07:40:37.301+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-02T07:40:37.305+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_raw_from_s3> on 2025-10-02 07:40:35.330938+00:00
[2025-10-02T07:40:37.308+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=486) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T07:40:37.309+0000] {standard_task_runner.py:63} INFO - Started process 488 to run task
[2025-10-02T07:40:37.310+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sentiment140_end2end_single', 'download_raw_from_s3', 'manual__2025-10-02T07:40:35.330938+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/edaproject.py', '--cfg-path', '/tmp/tmpyo_1ersm']
[2025-10-02T07:40:37.311+0000] {standard_task_runner.py:91} INFO - Job 83: Subtask download_raw_from_s3
[2025-10-02T07:40:37.334+0000] {task_command.py:426} INFO - Running <TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T07:40:35.330938+00:00 [running]> on host 9d9a5d472595
[2025-10-02T07:40:37.360+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sentiment140_end2end_single' AIRFLOW_CTX_TASK_ID='download_raw_from_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T07:40:35.330938+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-02T07:40:35.330938+00:00'
[2025-10-02T07:40:37.361+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T07:40:37.365+0000] {logging_mixin.py:188} INFO - [download_raw_from_s3] Using CSV_S3_URI=s3://298a1/sentiment140/***/main-data.csv
[2025-10-02T07:40:37.369+0000] {base.py:84} INFO - Using connection ID 'aws_default' for task execution.
[2025-10-02T07:40:37.369+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2025-10-02T07:40:55.726+0000] {logging_mixin.py:188} INFO - [download_raw_from_s3] Downloaded to /tmp/MAUD_train_raw.csv (238803811 bytes)
[2025-10-02T07:40:55.755+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-02T07:40:55.757+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T07:40:55.762+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sentiment140_end2end_single, task_id=download_raw_from_s3, run_id=manual__2025-10-02T07:40:35.330938+00:00, execution_date=20251002T074035, start_date=20251002T074037, end_date=20251002T074055
[2025-10-02T07:40:55.776+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T07:40:55.792+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-02T07:40:55.792+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
